app/lib/actions/katello/organization/manifest_refresh.rb

```diff
@@ -42,6 +42,13 @@ def rescue_strategy
         def humanized_name
           _("Refresh Manifest")
         end
+
+        def finalize
```

Comment:
this also does not call plan_self in `plan` method, so finalize is never triggered